website/models.py

```diff
@@ -1,72 +1,92 @@
 from django.db import models
+from datetime import date
+
 
 class Nav(models.Model):
-	"""Main navigation links"""
-	nav_name = models.CharField(max_length=200)
-	link = models.CharField(max_length=200)
-	position = models.IntegerField()
-	visible = models.BooleanField()
+    """Main navigation links"""
+    nav_name = models.CharField(max_length=200)
```

Comment:
Is it possible for u to change the name ```nav_name``` to simply ```name```

Reply:
No.